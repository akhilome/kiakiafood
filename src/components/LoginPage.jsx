import React from 'react';
import { Link } from 'react-router-dom';

const LoginPage = () => {
  return (
    <div>
      This is the login page! ğŸ”¥ See <Link to="/signup">signup</Link>
    </div>
  );
};

export default LoginPage;
