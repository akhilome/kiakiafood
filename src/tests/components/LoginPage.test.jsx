import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../../components/LoginPage';

describe('LoginPage', () => {
  it('should render LoginPage correctly', () => {
    const wrapper = shallow(<LoginPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
