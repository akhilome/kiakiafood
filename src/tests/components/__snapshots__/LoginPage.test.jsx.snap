// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage should render LoginPage correctly 1`] = `
<div>
  This is the login page.
  <br />
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
